pkg/network/multus_test.go

```diff
@@ -0,0 +1,147 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	osv1 "github.com/openshift/api/operator/v1"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("Testing multus", func() {
+	Describe("validateMultus", func() {
+		_validateMultus := func(
+			multusRequested bool,
+			openshiftNetworkOperatorRunning bool,
+			openshiftNetworkOperatorDisableMultiNetwork bool,
+		) []error {
+			conf := &opv1alpha1.NetworkAddonsConfigSpec{}
+			if multusRequested {
+				conf.Multus = &opv1alpha1.Multus{}
+			}
+
+			var openshiftNetworkConfig *osv1.Network
+			if openshiftNetworkOperatorRunning {
+				openshiftNetworkConfig = &osv1.Network{}
+				if openshiftNetworkOperatorDisableMultiNetwork {
+					newTrue := func() *bool {
```

Comment:
I probably just don't know Golang well but what's this construction for? Can't we just have it as a simple struct field?

Reply:
You cannot assign &true. You have to do this extra step to get pointer to bool. Don't know why, don't know about any other nicer way.